server/routes/handlers/assess/getDossier.ts

```diff
@@ -0,0 +1,21 @@
+import type { RequestHandler } from 'express'
+import { getDossier } from '../../../clients/manageRecallsApi/manageRecallsApiClient'
+import logger from '../../../../logger'
+
+export default function getDossierHandler(): RequestHandler {
+  return async (req, res, next) => {
+    const { recallId } = req.query
+    if (typeof recallId === 'undefined') {
+      logger.error('/get-dossier called without a recallId')
+      return res.sendStatus(400)
+    }
+    const base64EncodedPdf = await getDossier(recallId as string, res.locals.user.token)
+    const pdfContentsByteArray = Buffer.from(base64EncodedPdf.content, 'base64')
+
+    res.writeHead(200, {
+      'Content-Type': 'application/pdf',
+      'Content-Disposition': 'attachment; filename="dossier.pdf"',
```

Comment:
I believe we're opening. PDFs inline rather than forcing downloads.

Reply:
Copied this from the revocation order.  What should this be then?  Should it be the same for revocation order?

Reply:
It should use 'inline'. for content-disposition instead of 'attachment', same as. https://github.com/ministryofjustice/manage-recalls-ui/blob/main/server/routes/handlers/book/recallUploadDocuments.ts#L60
Yes revocation order should work the same

Reply:
Ok, I'll update both